### DISCUSSION TITLE
Creating a Plotnine Mondrian

### DISCUSSION THREAD

**Discussion by ddanieltan** [has2k1/plotnine#848] [2024 Plotnine Contest]
### Authors

Daniel Tan

### Links

[github](https://github.com/ddanieltan), [linkedin](https://www.linkedin.com/in/ddanieltan/)

### Full description

I saw Michael Chow's submission on using Plotnine to create art, which inspired me to try and recreate a Piet Mondrian-like piece using Plotnine. I share the code used to create this art piece and some learnings I made a long the way in this blogpost: **website redacted, url destination changed**

![42](https://github.com/user-attachments/assets/03bfbf21-3eec-4856-8e3c-8a79e90e7654)

### Code repository

https://github.com/ddanieltan/ddanieltan.com/tree/main/posts/plotnine-mondrian

**Comment by ghost** [comment]
This is a great project and a great explanation!

**Reply by ddanieltan** [reply]
Thank you!

**Comment by has2k1** [comment]
![cell-2-output-1](https://github.com/user-attachments/assets/e47920a7-f27a-4207-b8d5-d6dd1d00e8e8)

**Comment by EconomiCurtis** [comment]
Ahoy @ddanieltan , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@ddanieltan, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
